src/reducers/index.js

```diff
@@ -13,8 +13,18 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+      let newInc = state.count + 1;
+      return {
+        ...state,
+        count: newInc
+      }
```

Comment:
```
    case INCREMENT:
    // Fill in the body of this case
      let newInc = state.count + 1;
      return {
        ...state,
        count: newInc
      }
```

This works perfectly. Another way that you could have handled this, which would use less space, would be:

```
    case INCREMENT:
    // Fill in the body of this case
      return {
        ...state,
        count: state.count + 1;
      }
```